mono/mini/simd-intrinsics-netcore.c

```diff
@@ -258,10 +258,14 @@ emit_sys_numerics_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSigna
 		ins->type = STACK_I4;
 		return ins;
 	case SN_ConvertToInt32:
+		if (!COMPILE_LLVM (cfg))
+			return NULL;
 		etype = get_vector_t_elem_type (fsig->params [0]);
 		g_assert (etype->type == MONO_TYPE_R4);
 		return emit_simd_ins (cfg, mono_class_from_mono_type_internal (fsig->ret), OP_CVTPS2DQ, args [0]->dreg, -1);
 	case SN_ConvertToSingle:
+		if (!COMPILE_LLVM (cfg))
+			return NULL;
```

Comment:
These ones (SN_ConvertToInt32, SN_ConvertToSingle) are actually implemented in mini-x64 already. Thanks for fixing the rest!

Reply:
Feel free to leave them disabled I will follow up on this and add the lowering code eventually.